x-pack/legacy/plugins/rollup/public/visualize/editor_config.js

```diff
@@ -56,7 +56,7 @@ export function initEditorConfig() {
 
     // Set date histogram time zone based on rollup capabilities
     if (aggTypeName === 'date_histogram') {
-      const interval = fieldAgg.interval;
+      const interval = fieldAgg.fixed_interval ? fieldAgg.fixed_interval : fieldAgg.interval;
```

Comment:
Should this actually use the logic `fixed_interval || calendar_interval || interval`? https://github.com/elastic/kibana/issues/36306

Reply:
Yes, it should...